### PR TITLE
Updates on blog and workshops page on Layer5-ng

### PR DESCRIPTION
**Pull request by Jashpatel1**
Signed-off-by: Jashpatel1 <patel.5@iitj.ac.in>

**Description**

This PR makes updates on different pages on Layer5-ng.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/components/Card/index.js [hunk 1]
```diff
@@ -10,44 +10,44 @@ const Card = ({ frontmatter, fields }) => {
...
-                            see more <IoIosArrowRoundForward />
-                        </Link>
-                        {frontmatter.eurl && (
-                            <a className="readmore-btn" href={frontmatter.eurl}>
-                                <BiLinkExternal />
-                            </a>
-                        )}
+                    <div className="post-content-block">
+                        <h2 className="post-title">
+                            <Link className="anchor" to={fields.slug}>
```
Comment (Tanuj22) [inline src/components/Card/index.js:23]: There is no need for a `Link` tag above these when it's present on the entire card. Do this for other places

File: src/components/Card/index.js [hunk 2]
```diff
@@ -10,44 +10,44 @@ const Card = ({ frontmatter, fields }) => {
...
+                                {frontmatter.title}
+                            </Link>
+                        </h2>
+                        <div className="post-meta-block">
+                            {frontmatter.author && <span>By: {frontmatter.author}</span>}
+                            {frontmatter.date && (
+                                <>
+                                    <span className="divider">/</span>
```
Comment (Tanuj22) [inline src/components/Card/index.js:31]: Remove divider as discussed. The date can then be left aligned

File: src/templates/blog.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ export const query = graphql`
                 id
                 frontmatter {
                     title
-                    date(formatString: "Do MMMM YYYY")
+                    date(formatString: "MMMM Do, YYYY")
```
Comment (Tanuj22) [inline src/templates/blog.js:29]: Do this for news and any other places dates are used

File: src/components/Card/index.js [hunk 4]
```diff
@@ -10,44 +10,44 @@ const Card = ({ frontmatter, fields }) => {
...
-                            </a>
-                        )}
+                    <div className="post-content-block">
+                        <h2 className="post-title">
+                            <Link className="anchor" to={fields.slug}>
+                                {frontmatter.title}
+                            </Link>
+                        </h2>
+                        <div className="post-meta-block">
+                            {frontmatter.author && <span>By: {frontmatter.author}</span>}
```
Comment (Tanuj22) [inline src/components/Card/index.js:28]: The `By` was supposed to be removed right?

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
The arrow animation on `see more` should occur as soon as we hover on the card now and not when we go near it

File: src/components/Card/index.js (hunk 1 above)
Comment (Jashpatel1) [reply src/components/Card/index.js:23]: It was discussed to have the entire card clickable.

File: src/components/Card/index.js (hunk 4 above)
Comment (Jashpatel1) [reply src/components/Card/index.js:28]: I didn't find it in the doc. I will do that if required.

File: src/components/Card/index.js (hunk 1 above)
Comment (Tanuj22) [reply src/components/Card/index.js:23]: So just put a link tag on the entire card instead of individual places

**Comment by leecalcote** [comment]
You will want to double check on the blog post card in grid view, specifically the post date and the post author don’t seem to be on the same line (they are vertically misaligned).

**Comment by leecalcote** [comment]
Nearest I can tell him I’m on the phone, the workshops card is much improved.

**Comment by Nikhil-Ladha** [comment]
One thing, should we add something like `Read more` with a `down arrow` when the card is not expanded and then change it to just `^` arrow on expansion?
It would not be specially focused or designed button just like a text element or so?
For UX?

**Comment by Jashpatel1** [comment]
@Nikhil-Ladha added the arrow

**Comment by Nikhil-Ladha** [comment]
Why is the arrow pointed diagonally?
And also the text `Read more` is missing.

**Comment by Jashpatel1** [comment]
> Why is the arrow pointed diagonally?
> And also the text `Read more` is missing.
> One more thing, why ae we adding a `Green` border on the top of each card in the workshops page?

I thought as the card is expanding in both directions, so a diagonal arrow.
I will add the text.
I don't know about the border part. Will look into it.

**Comment by Nikhil-Ladha** [comment]
> I thought as the card is expanding in both directions, so a diagonal arrow.

Hmmm.....ypu are right at your point. But, not sure about this.
What do others have to say in this?
@Tanuj22 @leecalcote

**Comment by Tanuj22** [comment]
> > I thought as the card is expanding in both directions, so a diagonal arrow.
> 
> Hmmm.....ypu are right at your point. But, not sure about this.
> What do others have to say in this?
> @Tanuj22 @leecalcote

It makes sense logically but it's feeling kind of out of place. I would go with the downward pointing arrow 😅

**Review by Nikhil-Ladha — APPROVED** [review]
Nice! :100:

**Review by Tanuj22 — APPROVED** [review]
😆

**Pull request merged**